scripts/constants.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+# Copyright (C) 2023, Ava Labs, Inc. All rights reserved.
+# See the file LICENSE for licensing terms.
+
+# Use lower_case variables in the scripts and UPPER_CASE variables for override
+# Use the constants.sh for env overrides
+
+# Set the PATHS
+GOPATH="$(go env GOPATH)"
```

Comment:
thinking if we should have a `TELEPORTER_PATH` here, then the path can be sourced when we want to reuse a Teleporter script.